app/controllers/ctc/questions/email_verification_controller.rb

```diff
@@ -2,13 +2,13 @@ module Ctc
   module Questions
     class EmailVerificationController < QuestionsController
       include AnonymousIntakeConcern
+      before_action :redirect_if_duplicate_ctc_client
       before_action :send_verification_code, only: [:edit]
 
       layout "intake"
 
       # if the client already has a valid intake with same info, don't have them verify again.
```

Comment:
I think this comment line can & should now be deleted? Not a blocker, but would be really nice.